### PR TITLE
chore: split event app

### PR DESCRIPTION
**Pull request by Kelvin-M**
The idea is to separate consuming event between localrep (locasync) and compute engine.
Localsync has been added in server pod (in the future, localrep django models and backend server django models will be merged) and event app has been renamed worker-event-app.

To do :
- [x] rename to worker_last_event.py
- [x] backend folder names (localsync, events) don't match the helm deployment names (localsync-events, worker-events)
- [x] update CI log upload

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201927380638320

### CODE REVIEW

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Review by sergebouchut2 — APPROVED** [review]

File: charts/substra-backend/values.yaml [hunk 1]
```diff
@@ -449,52 +449,102 @@ containerRegistry:
...
+  enabled: true
+  ## @param api_events.image.registry Substra event app image registry
+  ## @param api_events.image.repository Substra event app image repository
+  ## @param api_events.image.tag Substra event app image tag
+  ## @param api_events.image.pullPolicy Substra event app image pull policy
+  ## @param api_events.image.pullSecrets Specify image pull secrets
+  ##
+  image:
+    registry: gcr.io
+    repository: connect-314908/connect-backend
```
Comment (mblottiere) [inline charts/substra-backend/values.yaml:516]:
```suggestion
    registry: ghcr.io
    repository: substra/substra-backend
```

File: charts/substra-backend/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 22.1.0
+
+### Changed
+- Split compute engine and localrep events consumers
```
Comment (mblottiere) [inline charts/substra-backend/CHANGELOG.md:6]: Maybe for consistency?
```suggestion
- Split compute engine and api events consumers
```

**Review by mblottiere — COMMENTED** [review]
Instead of having a separate `api_event` django app, wouldn't it make sense to have everything bundled in the `api` app?

**Comment by Kelvin-M** [comment]
> Instead of having a separate `api_event` django app, wouldn't it make sense to have everything bundled in the `api` app?

And do the same for `worker_event` bundled in the `substrapp` ?

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Review by AlexandrePicosson — APPROVED** [review]
LGTM

**Pull request merged**